### PR TITLE
support unicode comma in flip

### PR DESCRIPTION
**Pull request by songgao**

### CODE REVIEW

**Comment by songgao** [comment]
This finally passed. It flaked several times but I can't see how those can be affected by this change.

**Comment by songgao** [comment]
@mikem mind taking a look? Thanks!

File: go/chat/flipmanager.go [hunk 1]
```diff
@@ -704,10 +704,14 @@ func (m *FlipManager) parseMultiDie(arg string, nPlayersApprox int) (start flip.
 	return flip.NewStartWithBigInt(m.clock.Now(), val, nPlayersApprox), nil
 }
 
+const shuffleSeparaters = ",，"
```
Comment (mmaxim) [inline go/chat/flipmanager.go]: is there a space in there?

Comment (songgao) [reply go/chat/flipmanager.go]: Nuh; Unicode commas come with built-in spacing for free 😉

Comment (songgao) [reply go/chat/flipmanager.go]: bump!

**Pull request merged**